tests/acceptance/expectUserAndAdminLoginCest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/*
+ Cest is a class based format. Cest format is able to support multiple 
+ tests per file and easy reuse of code by adding additional private functions.
+ Cest combines scenario-driven test approach with OOP design. 
+ */
+class expectUserAndAdminLoginCest
+{
+    public function _before(AcceptanceTester $I)
+    {
+        //runs before each test
+        $I->amOnPage('/');
+    }
+    public function _after(AcceptanceTester $I)
+    {
+        //runs after each test
+    }
+    // tests
+    
+    public function expectsUserToLogin(AcceptanceTester $I) {
+        $I->wantTo('log in as a user');
+        $I->loginAs('chappress', 'password');
+    }
+    public function expectsAdminToLogin(AcceptanceTester $I) {
+        $I->wantTo('log in as an Admin');
+        $I->loginAsAdmin();
+        $I->amOnPage('/wp-admin/index.php');
+    }
+    function expectsAddNewUser(AcceptanceTester $I)
```

Comment:
Hey @spence-man, is there a reason why this is just `function` and not `public function` (or private)? It works properly and I saw an example of this [here](http://codeception.com/docs/07-AdvancedUsage#Dependency-Injection) in the last block of code of the Dependency Injection section. I'm just wondering for my own knowledge. Is it because it defaults to public (or private)?